scripts/add_content_to_db.py

```diff
@@ -70,17 +70,44 @@
     }
 
     with open(args.csv, mode="r", encoding="utf-8") as file:
+        reader = csv.DictReader(file)
+        tags = [val.upper() for row in reader for val in eval(row["tags"])]
+        tags_db = requests.get(tags_endpoint, headers=headers)
+        tags_map = {val["tag_name"]: val["tag_id"] for val in tags_db.json()}
+
+        tags_db = [val["tag_name"] for val in tags_db.json()]
+        tags = set(tags) - set(tags_db)
+        if len(tags) > 0:
+            print(f"Adding new tags: {tags}")
+        for tag in tags:
+            payload = {
+                "tag_name": tag,
+            }
+            response = requests.post(
+                tags_endpoint, json=payload, headers=headers, verify=False
```

Comment:
```suggestion
                tags_endpoint, json=payload, headers=headers
```
Should we add a comment here to also pass `verify=False` if you are running it locally?